package.json

```diff
@@ -187,7 +187,7 @@
     "rimraf": "^2.6.2",
     "sinon": "^4.1.2",
     "size-limit": "0.14.1",
-    "typescript": "^2.6.1",
+    "typescript": "2.6.2",
```

Comment:
Please revert.

Reply:
Ok, but again, it doesn't compile in 2.6.1.  Do you want me to remove the fragments from the test?

Reply:
That doesn't matter, the `yarn.lock` is what controls the actual version this project depends on, and it did point to `2.8.1` until you changed it. There should be no changes to either `package.json` or `yarn.lock` in this PR.